### PR TITLE
[Jest] Migrate `Load glyph range.test`

### PR DESCRIPTION
**Pull request by astridx**

### CODE REVIEW

**Review by HarelM — APPROVED** [review]

File: src/style/load_glyph_range.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    let request;
+    useFakeXMLHttpRequest().onCreate = (req) => { request = req; };
+
+    loadGlyphRange('Arial Unicode MS', 0, 'https://localhost/fonts/v1/{fontstack}/{range}.pbf', manager, (err, result) => {
+        expect(err).toBeFalsy();
+        expect(transform).toHaveBeenCalledTimes(1);
+        expect(transform).toHaveBeenCalledWith('https://localhost/fonts/v1/Arial Unicode MS/0-255.pbf', 'Glyphs');
+
+        if (!result) return done.fail(); // appease flow
```
Comment (wipfli) [inline src/style/load_glyph_range.test.ts:24]: remove `// appease flow`

**Review by wipfli — APPROVED** [review]
Thanks for migrating this Astrid. Would be good if you could clean up the flow comment before merging.

File: src/style/load_glyph_range.test.ts (hunk 1 above)
Comment (astridx) [reply src/style/load_glyph_range.test.ts]: Thanks for pointing that out and for checking it out. I had also thought about it. I wanted to delete the line altogether. If it's only there to make Flow happy, then I don't need it any more, right? But maybe I'm missing something?

Comment (wipfli) [reply src/style/load_glyph_range.test.ts]: We don't need them. Looks good to me.

**Pull request merged**